client/src/document/index.scss

```
@@ -942,3 +942,41 @@ kbd {
     opacity: 0.4;
   }
 }
+
+html a.only-in-en-us:after {
+  content: "(en-US)";
+  font-size: smaller;
+  vertical-align: super;
+}
+
+html[lang="es"] a.only-in-en-us:after {
+  content: "(inglés)";
```

@Graywolf9 what do you think on this one? I'd like to say the following:

```suggestion
  content: "(Inglés)";
```
Do you agree? or do you think it is better without the uppercase?

---

I already checked it, according [this resource](https://www.fundeu.es/consulta/minuscula-o-mayuscula-5755/#:~:text=Tal%20como%20indica%20la%20Ortografía,%2C%20francés%2C%20chino%2C%20alemán…) it shouldn't use uppercase for the first letter. so this is good!